azurecompute/src/main/java/org/jclouds/azurecompute/AzureManagementApiMetadata.java

```diff
@@ -57,16 +63,17 @@ public static Properties defaultProperties() {
       protected Builder() {
          id("azurecompute")
          .name("Microsoft Azure Service Management Service API")
-         .version("2014-06-01")
+         .version("2014-10-01")
          .identityName("Path to Management Certificate .p12 file, or PEM string")
          .credentialName("Password to Management Certificate")
          .defaultEndpoint("https://management.core.windows.net/${" + SUBSCRIPTION_ID + "}")
```

Comment:
Endpoints are meant to be usable *without* user modification. If there is no endpoint that can be hardcoded, just remove it.